### DISCUSSION TITLE
Add more precise tablet zone customization and tablet sensibility parameter

### DISCUSSION THREAD

**Discussion by YiraSan** [ppy/osu#23349] [Ideas]
I wanted to switch from osu!classic to osu!lazer but i'm not able to get the same sensibility/ratio zone as I configured it for osu!classic, making me unable to switch between both classic and lazer without having to learn again my aim.

I currently use a wacom CTL-4100WL to play with, my parameter on the wacom tablet properties are the following:
![image](https://user-images.githubusercontent.com/45128167/235361350-c6111aa7-2260-494c-87bf-7e351991f70c.png)
but unfortunatelly osu is limited to two digits precision making me unable to have the exact same ratio.

Also on osu!classic the mouse/tablet sensibility are merged making my able to have a 1.96x sensibility with this tablet zone configuration, but on osu!lazer, I can only configure a sensibility for the mouse... Maybe add a sensibility paramter only for tablets.

It would be amazing since I really want to switch from osu!classic to osu!lazer when the time will come for the next big update :)

**Comment by bdach** [comment]
https://github.com/ppy/osu/discussions/13188

**Reply by YiraSan** [reply]
I didn't see that, but I still can't change the tablet sensibility ?

**Reply by bdach** [reply]
yes, the point is that this is a duplicate of that linked discussion and nothing more. the textboxes will be added eventually

**Reply by YiraSan** [reply]
I first didn't wanted to reanswer because I don't want to disturb you too much, but the fact is that on osu!classic we can edit the sensibility of both the tablet and the mouse, my entire post wasn't duplication since it would be interesting to add a tablet sensibility like osu!classic on osu!lazer
